server/container/util-java8/src/test/java/org/apache/james/util/FluentFutureStreamTest.java

```diff
@@ -33,6 +36,23 @@
 public class FluentFutureStreamTest {
 
     @Test
+    public void test() throws IOException {
```

Comment:
test ?

Reply:
Oups. This was a debug statement to confirm execution order of cath close, and thus apply correctly MDC. Will remove.